src/Dibi/Drivers/FirebirdResult.php

```diff
@@ -20,7 +20,7 @@ class FirebirdResult implements Dibi\ResultDriver
 {
 	use Dibi\Strict;
 
-	/** @var resource */
+	/** @var resource|null */
 	private $resultSet;
```

Comment:
is never null

Reply:
I reverted and changed `getResultResource()` because it return always `resource`.

Reply:
Reverted.